### DISCUSSION TITLE
Supplementary Materials need to be Updated

### DISCUSSION THREAD

**Discussion by TullyMonster** [biostars/biostar-handbook-issues#299] [General]
[https://www.biostarhandbook.com/whatisdata.html - What kind of problems does bioinformatics data have?](https://www.biostarhandbook.com/whatisdata.html#:~:text=you%20heard%20that-,ten%20years%20later,-we%20realized%20that)

Time flies, it's almost 2024. [The human genome at 20](https://www.nature.com/collections/febigehfbg) has been published~
> Of course, it's okay not to supplement it.😄

**Comment by ialbert** [comment]
I have added the link you mention, it is quite useful.
